<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sołdaczuk_Dyśko_AK2</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/images/bmp_icon.png</normalon>
   </iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="podpis">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Goudy Old Style</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Marek Sołdaczuk / Tomasz Dyśko</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_start">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>186</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(180, 180, 180);
border: 1px solid gray;

    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}
QPushButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>Wczytaj obraz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_wyj">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>186</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(200, 200, 200);
border: 1px solid gray;

    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: red;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}
QPushButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>Zakończ</string>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1031</width>
      <height>771</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bitmapa">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Bitmapa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_e1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>186</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(180, 180, 180);
border: 1px solid gray;

    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}
QPushButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>Efekt 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_e2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>186</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(180, 180, 180);
border: 1px solid gray;

    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}
QPushButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>Efekt 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_e3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>186</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(180, 180, 180);
border: 1px solid gray;

    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}
QPushButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>Efekt 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_e4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>186</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(180, 180, 180);
border: 1px solid gray;

    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}
QPushButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>Efekt 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_e5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>186</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(180, 180, 180);
border: 1px solid gray;

    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}
QPushButton:pressed { background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>Efekt 5</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>231</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8514oem</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> border: 2px solid black;
    border-radius: 3px;
    padding: 2px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>background</zorder>
   <zorder>podpis</zorder>
   <zorder>but_start</zorder>
   <zorder>but_wyj</zorder>
   <zorder>bitmapa</zorder>
   <zorder>but_e1</zorder>
   <zorder>but_e2</zorder>
   <zorder>but_e3</zorder>
   <zorder>but_e4</zorder>
   <zorder>but_e5</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
